### PR TITLE
expose new names for methods

### PR DESCRIPTION
**Pull request by unity**
- Exposes `Hull.identify` as a segment-compatible method (alias for `Hull.track`)
- Exposes `hull.identify` as a valid message for the event bus
- Exposes `Hull.alias` as a valid pre-boot method
- Exposes `hjs_attr_*` as valid querystring parameters for tracking
- Updates docs to refer to `Hull.identify` instead of `Hull.traits` (backwards compatibility unaffected)
- Removes sections about Identity Management from Docs
- Reorganizes sections

### CODE REVIEW

File: src/client/traits/index.coffee [hunk 1]
```diff
@@ -38,6 +38,7 @@ module.exports = (api)->
         # });
       _.map name, (value, key)-> new Trait(api, key, value)
       Hull.emit('hull.traits',name)
+      Hull.emit('hull.identify',name)
```
Comment (michaloo) [inline src/client/traits/index.coffee]: does it mean we start to emit two things on `Hull.traits()` call?

Comment (unity) [reply src/client/traits/index.coffee]: we emit the same message on 2 channels

Comment (michaloo) [reply src/client/traits/index.coffee]: does it mean we can start producing higher volumes of incoming events?

Comment (unity) [reply src/client/traits/index.coffee]: this is the client-side Hull Event Emitter:  https://www.hull.io/docs/reference/hull_js/#event-emitter

Not related to tracking calls.

**Pull request merged**